laundry/api_wrapper.py

```diff
@@ -0,0 +1,200 @@
+import csv
+import os
+
+import pkg_resources
+import requests
+from bs4 import BeautifulSoup
+
+
+LAUNDRY_URL = os.environ.get("LAUNDRY_URL", "http://suds.kite.upenn.edu")
+ALL_URL = "{}/?location=".format(LAUNDRY_URL)
+
+# USAGE_BASE_URL from LAS does not work
+
+
+class Laundry(object):
```

Comment:
Depending on our discussion about a LaundryMachine model (see other comment) this may need to be rewritten slightly

Reply:
Okay, I think now that we have all of the information about laundry rooms stored in the DB, I think we refactor this so that we don't create a new class, but instead just provide functions that reference the DB instead of maintaining a duplicate set of information in memory